src/main/java/blackjack/domain/participant/Players.java

```diff
@@ -0,0 +1,89 @@
+package blackjack.domain.participant;
+
+import blackjack.domain.card.Deck;
+import blackjack.domain.game.PlayerResult;
+import blackjack.domain.game.Referee;
+import blackjack.domain.game.result.GameResult;
+import java.util.List;
+
+public class Players {
+
+    private static final int MAX_PLAYERS_SIZE = 4;
+
+    private final Dealer dealer;
+    private final List<Player> players;
+
+    private Players(Dealer dealer, List<Player> players) {
+        this.dealer = dealer;
+        validateSize(players);
+        validateDistinct(players);
+        this.players = players;
+    }
+
+    public static Players from(List<String> names) {
+        List<Player> players = names.stream()
+                .map(Player::from)
+                .toList();
+        return new Players(new Dealer(), players);
+    }
+
+    private void validateSize(List<Player> players) {
+        if (players.isEmpty()) {
+            throw new IllegalArgumentException("최소 한 명의 플레이어가 있어야 합니다.");
+        }
+        if (players.size() > MAX_PLAYERS_SIZE) {
+            throw new IllegalArgumentException("최대 4명의 플레이어만 참여 가능합니다.");
+        }
+    }
+
+    private void validateDistinct(List<Player> players) {
```

Comment:
조금 더 나은 네이밍이 떠올라서 적어 두겠습니다 😁
```suggestion
    private void validateUniquePlayerName(List<Player> players) {
```

Reply:
👍

Reply:
`Name`이 `Player`에게 있을 거라는 생각을 못 할 수 있지 않을까? 라는 생각에 적었지만, 충분히 도출 가능하다고 판단하면 짱수 의견대로 해도 괜찮을 듯하네요!